src/views/boost/l10n.json

```diff
@@ -0,0 +1,34 @@
+{
+    "boost.headerText": "The {boostLink} kit brings your LEGO creations to life with powerful motors, a color sensor and more. By combining it with Scratch, you can build your own robotic creatures, tell physical-digital stories, invent new game controllers, or whatever else you can imagine.",
+    "boost.gettingStarted": "Getting Started",
+    "boost.connectingBoost": "Connecting BOOST to Scratch",
+    "boost.powerBoost": "Turn on your sensor by pressing the power button.",
+    "boost.useScratch3": "Use the {scratch3Link} editor.",
+    "boost.addExtension": "Add the BOOST extension.",
+    "boost.thingsToTry": "Things to Try",
+    "boost.makeAMotorMove": "Make a motor move",
+    "boost.findTurnMotorOnForSeconds": "Find the {turnMotorOnForSeconds} block and click on it.",
+    "boost.turnMotorOnForSeconds": "“turn motor A on for 1 seconds”",
+    "boost.connectALegoBeam": "Connect a LEGO beam with an axle to motor A and click the block again to make it spin.",
+    "boost.starterProjects": "Starter Projects",
+    "boost.troubleshootingTitle": "Troubleshooting",
+    "boost.checkOSVersionTitle": "Make sure your operating system is compatible with Scratch Link",
+    "boost.checkOSVersionText": "The minimum operating system versions are listed at the top of this page. See instructions for checking your version of {winOSVersionLink} or {macOSVersionLink}.",
+    "boost.winOSVersionLinkText": "Windows",
+    "boost.macOSVersionLinkText": "Mac OS",
+    "boost.closeScratchCopiesTitle": "Close other copies of Scratch",
```

Comment:
I think ideally we would move some of these common strings -- "Starter Projects", etc. -- into a shared general file, rather than reproducing them for each extension. But it's no worse here than with 5 other extensions' l10n files, so I guess this is fine for now.

Reply:
Good point! As we keep adding more of these landing pages, it might be time for some cleanup of the strings... I'll file it as a follow up issue.